app/javascript/modules/listingDetails/ListingDetailsReservedBanner.tsx

```diff
@@ -1,33 +1,60 @@
 import React from "react"
 import { Message, t } from "@bloom-housing/ui-components"
 import { RESERVED_COMMUNITY_TYPES } from "../constants"
+import { getCustomListingType, renderInlineMarkup } from "../../util/languageUtil"
 
 export interface ListingDetailsReservedBannerProps {
   reservedCommunityMinimumAge?: number
   reservedCommunityType?: string
+  customListingType?: string
 }
 
-export const ListingDetailsReservedBanner = ({
-  reservedCommunityMinimumAge,
-  reservedCommunityType,
-}: ListingDetailsReservedBannerProps) => {
-  if (!reservedCommunityType || reservedCommunityType === RESERVED_COMMUNITY_TYPES.HABITAT)
-    return null
+const buildMessage = (
+  reservedCommunityMinimumAge?: number,
+  reservedCommunityType?: string,
+  customListingType?: string
+): React.JSX.Element => {
+  if (customListingType && getCustomListingType(customListingType)) {
+    return (
+      <Message warning={true}>
+        {renderInlineMarkup(`${t("listings.customListingType.educator.banner")}`, "<p><b>")}
+        {renderInlineMarkup(
+          t(`listings.customListingType.educator.banner.link.content`, {
+            url: `#listing-details-eligibility`,
+          }),
+          "<p><a>"
+        )}
+      </Message>
```

Comment:
Correct me if I'm wrong, but It looks like a lot of work has been done to work around the limitations of the Bloom `Message` component, which will normally display text in all-caps with no line breaks. E.g. Adding HTML tags into translations and using the override feature of the `markdown-to-jsx` package.

I think the work arounds are fine, in the interest of time, but we should add some comments to indicate that these code changes are workarounds, and also make a feature request to Bloom developers, to make their `Message` component more flexible.

Reply:
Yes partly. Today the `Message` component does take in React components and respects things like break lines, anchor tags, and bold text. But it does by default make everything capitalized. I reverted updating the `Markdown` component and added the styling to the html itself to force `Message` to respect the non capitalized letters. I think there also may be a way to override some of the classNames of `Message` that force it to be all capital, I'll take a look and if not we can open a request with Bloom.

Reply:
Hmm, updating the `Markdown` component and adding inline styles to translation files both seem equally unideal to me, but are fine as short term workarounds. I'll leave it up to you to decide which way is better.